readme.md

```diff
@@ -679,24 +679,22 @@ New versions of the if and switch statements for C++: `if (init; condition)` and
 This should simplify the code. For example, previously you had to write:
 
 ```cpp
-{   
-    auto val = GetValue();   
-    if (val)    
-        // on success  
-    else   
-        // on false... 
-}
+int val = GetValue();
```

Comment:
'auto' serves better, because it can mean any type... I don;t want to go into 'int'.
Also the surrounding clause { } is needed, otherwise 'val' will leak outside, as I wrote it.